docker/mlclustering/Pipfile.lock

```diff
@@ -165,13 +165,16 @@
                 "sha256:873d13d177501a28b2756375d59816c365e42ed8417b41665f346289adc68d24",
                 "sha256:8e5a0b00e1e56a842f922e7fae8ae4077aee4af0acb5ae3622bd4b4c30aedf99",
                 "sha256:8e90497254aacacbc4ea6ae5e7a8cd75629d6ad2b30025a4a8b09aa4faf55151",
+                "sha256:9057e6aa78a584bc93a13f0a9bf7e753a5e9770a30b4d758b8d5f2a62a9433cd",
```

Comment:
is every other library already up to date in this image?

Reply:
No, I don't know why `pipenv` is acting this way but most if not all of the libraries were updated.

Reply:
so lets run pipenv lock to make sure its up to date

Reply:
I did